src/components/icon/icon.js

```diff
@@ -65,141 +94,148 @@ class Icon extends Component {
    * @memberof Icon
    */
   render(props: any): React$Element<any> | void {
-    switch (props.type) {
-      case IconType.Maximize:
-        return <i className={[style.icon, style.iconMaximize].join(' ')} />;
+    if (props.path) {
+      const backgroundStyle = {
+        backgroundImage: this.svgUrl(props.path)
+      };
+      return <i style={backgroundStyle} className={style.icon} />;
+    } else {
+      switch (props.type) {
```

Comment:
I think it'll be better to move those icons to the relevant components

Reply:
In the next phase. 
Want to keep this PR simple and first to support external icons.
Later to do it for all the rest that currently "lives" inside the ui.